spec/integration/support/inspect_unmanaged_files_examples.rb

```diff
@@ -63,6 +63,7 @@ def parse_md5sums(output)
 
       expected = <<EOF
 Inspecting unmanaged-files...
+Note: Using traditional inspection because there is no helper binary for architecture 'x0_0' available.
```

Comment:
If it's too much work it's not worth the effort. A comment explaining that `normalize_inspect_output` adapts the text which leads to `x0_0` could help future developers when they'll read the tests.